app/services/image_upload_requirements.rb

```diff
@@ -19,7 +21,7 @@ def errors
       return [I18n.t!("document_images.index.flashes.upload_requirements.no_file_selected")]
     end
 
-    if SUPPORTED_FORMATS.exclude?(Marcel::MimeType.for(file))
+    if SUPPORTED_FORMATS.exclude?(Marcel::MimeType.for(file)) || MiniMagick::Image.new(file.tempfile.path).frames.count > 1
```

Comment:
Maybe make this a separate if block, as it's quite specific/long?

Reply:
Yep it is a bit long, i just thought that was better than repeating code with another if when both blocks would be concerning invalid formats?

Reply:
I could move it into a method with a descriptive name which would make the line shorter and clearer